lib/msal-browser/src/utils/BrowserConstants.ts

```diff
@@ -81,6 +81,19 @@ export enum InteractionType {
     Silent = "silent"
 }
 
+/**
+ * Type of interaction in progress
```

Comment:
I think it would be good to provide more details on this enum, potentially including annotating each value.

Reply:
Updated